apps/cookbook/src/app/showcase/label-showcase/label-showcase.component.html

```diff
@@ -0,0 +1,27 @@
+<h2>Label Showcase</h2>
```

Comment:
Can probably be removed.
```suggestion
<h2>Label Showcase</h2>
```